lib/binary_to_decimal.rb

```diff
@@ -1,9 +1,11 @@
-# A method named `binary_to_decimal` that receives as input an array of size 8.
-# The array is randomly filled with 0’s and 1’s.
-# The most significant bit is at index 0.
-# The least significant bit is at index 7.
-# Calculate  and return the decimal value for this binary number using
-# the algorithm you devised in class.
+
 def binary_to_decimal(binary_array)
-  raise NotImplementedError
-end
+  sum = 0
+  decimal_array =  [128, 64, 32, 16, 8, 4, 2, 1]
```

Comment:
This works, but you've hardcoded for specific powers of 2.  Could you find a way to make this more flexible?